docs/content/en/docs/Reference/auroraboot.md

```diff
@@ -37,6 +37,19 @@ AuroraBoot can be useful to:
 - `ProxyDHCP` supported by the `DHCP` network attempting to netboot (see also [pixiecore architecture](https://github.com/danderson/netboot/blob/master/pixiecore/README.booting.md#step-1-dhcpproxydhcp)).
    There should be an already running `DHCP` server on your network. AuroraBoot doesn't take over the `DHCP` server, neither require you to do any specific configuration, however a `DHCP` server which is compliant to `ProxyDHCP` requests should be present in the same network running **AuroraBoot** and the machines to boot.
 
+## MacOS
+
+Unfortunately for macOS systems we cannot run the netboot trough docker as its being run inside a VM, so it cant see the host network.
+Building ISO still works as long as you mount the container `/tmp` disk to a local dir so its exported there.
```

Comment:
maybe we can provide a full(tested) example here to copy-paste?

Reply:
good point